Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```
@@ -11,10 +11,22 @@
   box-sizing: border-box;
 }
 
+.tabStripTabsPane ul::after
+{
+  content: "";
+  flex-grow: 99999;
+}
+
+.tabStripTabsPane ul li
+{
+  flex-grow: 1;
+}
+
 .tabStripTabWrapper .tabStripTab,
```

noticed this during testing: please add a fix-commit for tabstrip to make the tabs use the standard white background.

```suggestion

.tabStripTabWrapper .tabStripTab
{
  background-color: var(--color-standard-background);
}

.tabStripTabWrapper .tabStripTab,
```